Content/src/Client/Client.fs

```diff
@@ -159,10 +163,40 @@ let update (msg : Msg) (currentModel : Model) : Model * Cmd<Msg> =
     | _, InitialCountLoaded initialCount->
         let nextModel = { Counter = Some initialCount }
         nextModel, Cmd.none
-
     | _ -> currentModel, Cmd.none
 #endif
 
+#if (reaction)
+
```

Comment:
those will result in many empty lines in generated project, let's try to minimise them